From f8da985866d1411346067d125c3a0f8b1232606d Mon Sep 17 00:00:00 2001
From: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
Date: Mon, 14 Sep 2020 15:25:11 +0530
Subject: [PATCH 45/45] Set GPIO_0(EIM_A25) to 100K ohm pull up

This patch will mux PAD_EIM_A25 as gpio GPIO5_IO02 and set 100k ohm as
pull up.

Signed-off-by: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
---
 arch/arm/boot/dts/lec-imx6.dtsi | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/lec-imx6.dtsi b/arch/arm/boot/dts/lec-imx6.dtsi
index 833c759a6fff..0d90724a3451 100644
--- a/arch/arm/boot/dts/lec-imx6.dtsi
+++ b/arch/arm/boot/dts/lec-imx6.dtsi
@@ -489,12 +489,6 @@
 	status = "okay";
 };
 
-&hdmi_cec {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_hdmi_cec>;
-	status = "okay";
-};
-
 &hdmi_core {	/* HDMI (see fb above) */
 	ipu_id = <0>;
 	disp_id = <1>;
@@ -826,6 +820,7 @@
 				MX6QDL_PAD_NANDF_CS2__GPIO6_IO15 0x80000000 /* SMARC GPIO2/CAM0_RST# */
 				MX6QDL_PAD_NANDF_CS3__GPIO6_IO16 0x80000000 /* SMARC GPIO3/CAM1_RST# */
 				MX6QDL_PAD_NANDF_CLE__GPIO6_IO07 0x80000000 /* SMARC GPIO7/PCAM_FLD */
+				MX6QDL_PAD_EIM_A25__GPIO5_IO02  0x0b0b1 /* GPIO_0 set to 100k ohm pull up */
 			>;
 		};
 
@@ -928,12 +923,6 @@
 			>;
 		};
 
-		pinctrl_hdmi_cec: hdmicecgrp {
-			fsl,pins = <
-				MX6QDL_PAD_EIM_A25__HDMI_TX_CEC_LINE	0x108b0
-			>;
-		};
-
 		pinctrl_hdmi_hdcp: hdmihdcpgrp {
 			fsl,pins = <
 				MX6QDL_PAD_KEY_COL3__HDMI_TX_DDC_SCL 0x4001b8b1
-- 
2.17.1

